<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Rafael  Peres Da Silva | publications</title>
<meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.xyz/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->

<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>ðŸ”Ž</text></svg>">

<link rel="stylesheet" href="/assets/css/main.css">

<link rel="canonical" href="/publications/">

<!-- JQuery -->
<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>


<!-- Theming-->

  <script src="/assets/js/theme.js"></script>
  <!-- Load DarkMode JS -->
<script src="/assets/js/dark_mode.js"></script>



  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-1208335-9"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'UA-1208335-9');
  </script>




    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="/">
       <span class="font-weight-bold">Rafael</span>   Peres Da Silva
      </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">
              about
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item active">
              <a class="nav-link" href="/publications/">
                publications
                
                <span class="sr-only">(current)</span>
                
              </a>
          </li>
          
          
          
          
          
            <div class = "toggle-container">
              <a id = "light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
              </a>
            </div>
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h1 class="post-title">publications</h1>
    <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p>
  </header>

  <article>
    <div class="publications">


  <h2 class="year">2021</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">ML/Comp. Biology</abbr>
    
  
  </div>

  <div id="dasilva2020tugda" class="col-sm-8">
    
      <div class="title">TUGDA: Task uncertainty guided domain adaptation for robust generalization of cancer drug response prediction from in vitro to in vivo settings</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Silva, Rafael Peres,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Suphavilai, Chayaporn,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Nagarajan, Niranjan
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Bioinformatics (Proceedings of Intelligent Systems for Molecular Biology and European Conference on Computational Biology)</em>
      
      
        2021
      
      </div>
    

    <div class="links">
    
    
    
      <a href="https://www.biorxiv.org/content/10.1101/2020.12.17.415737v1" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
      <a href="http://github.com/csb5/TUGDA/" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Machine Learning</abbr>
    
  
  </div>

  <div id="dasilva2020task" class="col-sm-8">
    
      <div class="title">Task Uncertainty Loss Reduce Negative Transfer in Asymmetric Multi-task Feature Learning</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Silva, Rafael Peres,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Suphavilai, Chayaporn,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Nagarajan, Niranjan
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Proceedings of the AAAI Conference on Artificial Intelligence (Student Track)</em>
      
      
        2021
      
      </div>
    

    <div class="links">
    
    
    
      <a href="https://arxiv.org/abs/2012.09575" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li></ol>

  <h2 class="year">2020</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">ML/Comp. Biology</abbr>
    
  
  </div>

  <div id="Suphavilai2020.11.23.389676" class="col-sm-8">
    
      <div class="title">Predicting heterogeneity in clone-specific therapeutic vulnerabilities using single-cell transcriptomic signatures</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Suphavilai, Chayaporn,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Chia, Shumei,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Sharma, Ankur,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Tu, Lorna,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Da Silva, Rafael Peres,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Mongia, Aanchal,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  DasGupta, Ramanuj,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Nagarajan, Niranjan
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>bioRxiv</em>
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.biorxiv.org/content/10.1101/2020.11.23.389676v1" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
      <a href="https://github.com/CSB5/CaDRReS-sc" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>While understanding heterogeneity in molecular signatures across patients underpins precision oncology, there is increasing appreciation for taking intra-tumor heterogeneity into account. Single-cell RNA-seq (scRNA-seq) technologies have facilitated investigations into the role of intra-tumor transcriptomic heterogeneity (ITTH) in tumor biology and evolution, but their application to in silico models of drug response has not been explored. Based on large-scale analysis of cancer omics datasets, we highlight the utility of ITTH for predicting clinical outcomes. We then show that heterogeneous gene expression signatures obtained from scRNA-seq data can be accurately analyzed (80%) in a recommender system framework (CaDRReS-Sc) for in silico drug response prediction. Patient-derived cell lines capturing transcriptomic heterogeneity from primary and metastatic tumors were used as in vitro proxies for validating monotherapy predictions (Pearson r&gt;0.6), as well as optimal drug combinations to target different subclonal populations (&gt;10% improvement). Applying CaDRReS-Sc to the increasing number of publicly available tumor scRNA-seq datasets can serve as an in silico screen for further in vitro and in vivo drug repurposing studies.HighlightsLarge-scale analysis to establish the impact of transcriptomic heterogeneity within tumors on clinical outcomesCalibrated recommender system for drug response prediction based on single-cell RNA-seq data (CaDRReS-Sc)Prediction of drug response in patient-derived cell lines with transcriptomic heterogeneityIn silico identification of drug combinations that work based on clonal vulnerabilitiesCompeting Interest StatementThe authors have declared no competing interest.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2019</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Neuroscience</abbr>
    
  
  </div>

  <div id="doi:10.1002/cne.24518" class="col-sm-8">
    
      <div class="title">Neuronal response properties across cytochrome oxidase stripes in primate V2</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Peres, Rafael,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Soares, Juliana G. M.,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Lima, Bruss,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Fiorani, Mario,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Chiorri, Marco,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Florentino, Maria M.,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Gattass, Ricardo
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Journal of Comparative Neurology</em>
      
      
        2019
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/cne.24518" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Abstract Cytochrome oxidase histochemistry reveals large-scale cortical modules in area V2 of primates known as thick, thin, and interstripes. Anatomical, electrophysiological, and tracing studies suggest that V2 cytochrome oxidase stripes participate in functionally distinct streams of visual information processing. However, there is controversy whether the different V2 compartments indeed correlate with specialized neuronal response properties. We used multiple-electrode arrays (16â€‰Ã—â€‰2, 8â€‰Ã—â€‰4 and 4â€‰Ã—â€‰4 matrices) to simultaneously record the spiking activity (Nâ€‰=â€‰190 single units) across distinct V2 stripes in anesthetized and paralyzed capuchin monkeys (Nâ€‰=â€‰3 animals, 6 hemispheres). Visual stimulation consisted of moving bars and full-field gratings with different contrasts, orientations, directions of motion, spatial frequencies, velocities, and color contrasts. Interstripe neurons exhibited the strongest orientation and direction selectivities compared to the thick and thin stripes, with relatively stronger coding for orientation. Additionally, they responded best to higher spatial frequencies and to lower stimulus velocities. Thin stripes showed the highest proportion (80%) of neurons selective to color contrast (compared to 47% and 21% for thick and interstripes, respectively). The great majority of the color selective cells (86%) were also orientation selective. Additionally, thin stripe neurons continued to increase their firing rate for stimulus contrasts above 50%, while thick and interstripe neurons already exhibited some degree of response saturation at this point. Thick stripes best coded for lower spatial frequencies and higher stimulus velocities. In conclusion, V2 CytOx stripes exhibit a mixed degree of segregation and integration of information processing, shedding light into the early mechanisms of vision.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2018</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Neuroscience</abbr>
    
  
  </div>

  <div id="10.3389/fnbeh.2018.00166" class="col-sm-8">
    
      <div class="title">Frontal Alpha Asymmetry and Theta Oscillations Associated With Information Sharing Intention</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Fischer, Nastassja L.,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Peres, Rafael,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Fiorani, Mario
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Frontiers in Behavioral Neuroscience</em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.frontiersin.org/article/10.3389/fnbeh.2018.00166" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Social media has gained increasing importance in many aspects of everyday life, from building relationships to establishing collaborative networks between individuals worldwide. Sharing behavior is an essential part of maintaining these dynamic networks. However, the precise neural factors that could be related to sharing behavior in online communities remain unclear. In this study, we recorded electroencephalographic (EEG) oscillations of human subjects while they were watching short videos. The subjects were later asked to evaluate the videos based on how much they liked them and whether they would share them. We found that, at the population level, subjects watching videos that would not be shared had higher power spectral density (PSD) amplitudes in the theta band (4â€“8 Hz), primarily over the frontal and parietal sites of the right hemisphere, than subjects watching videos that would be shared. Previous studies have associated task disengagement with an increase in scalp-wide theta activation, which can be interpreted as a mind-wandering effect. This might suggest that the decision to not share the video may lead to a more automatic/effortless neural pattern. We also found that watching videos that would be shared was associated with lower PSD amplitudes in the alpha band (8â€“12 Hz) over the central and right frontal sites, and with more negative scores of frontal alpha asymmetry (FAA) index scores. These results may be related to previous work linking right-sided frontal EEG asymmetry to the pursuit of social conformity and avoidance of negative outcomes, such as social isolation. Finally, using support vector machine (SVM) algorithms, we show that these EEG parameters and preference rating scores can be used to improve the predictability of sharing information behavior. The information sharing-related EEG pattern described here could therefore improve our understanding of the neural markers associated with sharing behavior and contribute to studies about stimuli propagation.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2017</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="10.1007/978-3-319-74433-9_17" class="col-sm-8">
    
      <div class="title">Named Entity Recognition in Twitter Using Images and Text</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Esteves, Diego,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Peres, Rafael,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Lehmann, Jens,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Napolitano, Giulio
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>The International Conference on Web Engineering (ICWE)</em>
      
      
        2017
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://link.springer.com/chapter/10.1007/978-3-319-74433-9_17" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Named Entity Recognition (NER) is an important subtask of information extraction that seeks to locate and recognise named entities. Despite recent achievements, we still face limitations with correctly detecting and classifying entities, prominently in short and noisy text, such as Twitter. An important negative aspect in most of NER approaches is the high dependency on hand-crafted features and domain-specific knowledge, necessary to achieve state-of-the-art results. Thus, devising models to deal with such linguistically complex contexts is still challenging. In this paper, we propose a novel multi-level architecture that does not rely on any specific linguistic resource or encoded rule. Unlike traditional approaches, we use features extracted from images and text to classify named entities. Experimental tests against state-of-the-art NER for Twitter on the Ritter dataset present competitive results (0.59 F-measure), indicating that this approach may lead towards better NER models.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="10.1145/3148011.3154478" class="col-sm-8">
    
      <div class="title">Bidirectional LSTM with a Context Input Window for Named Entity Recognition in Tweets</div>
      <div class="author">
        
          
          
          
          
          
          
            
              
                
                  Peres, Rafael,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  Esteves, Diego,
                
              
            
          
        
          
          
          
          
          
          
            
              
                
                  and Maheshwari, Gaurav
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>K-CAP</em>
      
      
        2017
      
      </div>
    

    <div class="links">
    
    
    
      <a href="https://doi.org/10.1145/3148011.3154478" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li></ol>


</div>

  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2025 Rafael  Peres Da Silva.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank">Unsplash</a>.

    
    
  </div>
</footer>



  </body>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


</html>
